app/models/programme.rb

```diff
@@ -141,6 +141,24 @@ def snomed_target_disease_term
     SNOMED_TARGET_DISEASE_TERMS.fetch(type)
   end
 
+  SNOMED_TARGET_DISEASE_NAMES = { "flu" => "FLU" }.freeze
+
+  def snomed_target_disease_name
+    SNOMED_TARGET_DISEASE_NAMES.fetch(type)
+  end
+
+  WRITE_TO_IMMUNISATIONS_API_TYPES = %w[flu hpv].freeze
+
+  def can_write_to_immunisations_api?
+    type.in?(WRITE_TO_IMMUNISATIONS_API_TYPES)
+  end
```

Comment:
```suggestion
  def can_write_to_immunisations_api? = hpv? || flu?
```

If we're not using the `WRITE_TO_IMMUNISATIONS_API_TYPES` constant elsewhere, I think my suggest results in more readable code at a glance.

Reply:
Ooh yes, very tidy